Makefile

```diff
@@ -1,29 +1,33 @@
 SHELL := /bin/bash
+GO_BUILD_OPTION := -trimpath -tags netgo
 
 .PHONY: all check format lint build test generate tidy
 
 help:
 	@echo "Please use \`make <target>\` where <target> is one of"
 	@echo "  check               to do static check"
-	@echo "  build               to create bin directory and build"
+	@echo "  build               to create bin directory and build beyond-ftp"
 	@echo "  generate            to generate code"
 	@echo "  test                to run test"
 
-check: vet
+check: format vet
 
 format:
 	go fmt ./...
-
-vet:
-	go vet ./...
+	@echo "ok"
 
 generate:
-	@echo "generate code"
 	go generate ./...
-	go fmt ./...
+	@echo "ok"
```

Comment:
Do we really need an `ok` or `done` here?

Reply:
From my perspective, an `ok` here makes me feel secure to know it works well.

But I can accept to remove them :)